### ISSUE TITLE
30s after start, status is still unknown in the menu

### ISSUE THREAD

**Issue by guillaumerose** [code-ready/tray-macos#62]
During startup, the status is still unknown. It should probably say Starting or so.

```
$ crc status
CRC VM:          Running
OpenShift:       Unreachable (v4.6.15)
Disk Usage:      9.839GB of 32.72GB (Inside the CRC VM)
Cache Usage:     16.37GB
Cache Directory: /Users/guillaumerose/.crc/cache
```

Detailed Status window is working fine although there is a lag. It shows Unknown for couple of seconds before changing to Running.

**Comment by anjannath** [comment]
Yeah, this needs to be improved, right now only three states exists ("Running", "Stopped" and "Unknown"), the running, stopping etc. states are not used in the tray.

not sure what can be done for the lag though! as that lag is coming from fetching the status
